ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/AdminNodeInfo.java

```diff
@@ -73,6 +77,8 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
     final Map<String, Object> response = new HashMap<>();
     final Map<String, Integer> ports = new HashMap<>();
 
+    System.out.println("MRW: Returning node info");
```

Comment:
remove debugging

Reply:
Bah, thanks, will remove now and add the changelog entry.

Reply:
Both now sorted @macfarla